src/fides/api/service/connectors/rds_mysql_connector.py

```diff
@@ -0,0 +1,153 @@
+from functools import cached_property
+from typing import Any, Dict, List, Optional
+
+from botocore.exceptions import ClientError
+from loguru import logger
+from sqlalchemy.engine import Engine, LegacyCursorResult  # type: ignore
+from sqlalchemy.orm import Session
+
+from fides.api.common_exceptions import ConnectionException
+from fides.api.graph.execution import ExecutionNode
+from fides.api.models.connectionconfig import ConnectionTestStatus
+from fides.api.models.policy import Policy
+from fides.api.models.privacy_request import PrivacyRequest, RequestTask
+from fides.api.schemas.connection_configuration.connection_secrets_rds_mysql import (
+    RDSMySQLSchema,
+)
+from fides.api.service.connectors.query_config import MySQLQueryConfig, SQLQueryConfig
+from fides.api.service.connectors.rds_connector_mixin import RDSConnectorMixin
+from fides.api.service.connectors.sql_connector import SQLConnector
+from fides.api.util.collection_util import Row
+
+CA_CERT_URL = "https://truststore.pki.rds.amazonaws.com/global/global-bundle.pem"
+
+
+class RDSMySQLConnector(RDSConnectorMixin, SQLConnector):
+    """
+    Connector specific to RDS MySQL databases
+    """
+
+    secrets_schema = RDSMySQLSchema
+    namespace_meta: Optional[dict] = None
```

Comment:
This should be defined inside `__init__` if each connector instance is meant to have it's own value

```
def __init__(self):
    super().__init__()
    # Instance-level attribute (unique to each instance)
    self.namespace_meta: Optional[Dict] = None
```